### ISSUE TITLE
EM Feedback on descriptions for version 1.4.0-dev.9

### ISSUE THREAD

**Issue by emily-phet** [phetsims/ohms-law#80]
Just a few bits of feedback for @terracoda when she returns. I was reviewing this [link](http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-dev.9/ohms-law-a11y-view.html?accessibility).

Scene Summary
- Consider: "In this sim, the Play Area contains the Ohm's Law equation, V equals I times R, and a circuit. There are also sliders to play with voltage and resistance." Reason: I think it's useful to emphasize that the main objects of focus are the equation and the circuit - which I think having these two together up front and in their own sentence can help with. It is important that the user know there are also sliders, but having them in their own distinct sentence I think helps them stand out as a separate thing (not what to focus on conceptually, but what to interact with).
- Consider "Dots in the resistor indicate the amount of resistance." In general, I think it's good to avoid the use of "represent" in descriptions. 

State of Sim
- Consider "slightly" instead of "somewhat". "Slightly" feels more like "just a little bit different" while "somewhat" feels more ambiguous to me.
- Current arrows relative description doesn't seem to update (stays at "tiny arrows represent..."). 

The Circuit
- let's replace "represent" with "indicate" throughout.
- Sometimes in the description for the dots "number" is misspelled as "numbr"
- Consider "many" instead of "good many". I don't think "good many" is a commonly used phrase in the US (at least not any region that I've lived). So, I interpret this as "a lot". I think "many" would have the same or similar interpretation, without potentially being ambiguous for those not familiar with the phrase.

Sliders
- Would "Move" sliders instead of "change" be too explicit of a reference to the visual representation? Change sliders to explore changes seems a bit redundant, but that's probably fine if it's more commen to refer to "changing" sliders than "moving" sliders.

**Comment by jessegreenberg** [comment]
> Sometimes in the description for the dots "number" is misspelled as "numbr"

My fault, sorry about that.

**Comment by terracoda** [comment]
Regarding EM's comments in https://github.com/phetsims/ohms-law/issues/80#issue-252024628, I think all the feedback is good. I will update design doc to reflect wording changes in Scene Summary, State of Sim, and The Circuit.

Regarding, the last item about the sliders, "Move" or "Adjust" are both a little more directive than "Change", but I agree that they sound a little more accurate when directly describing how to use a slider control. We could also try a more indirect approach by NOT stating exactly what to do with the sliders. 

For example,
- "Explore changes in equation and circuit as voltage and resistance change."
- "Explore changes in sim as voltage and resistance change."
- "Explore changes as voltage and resistance change."

Or, if we think we should keep the word slider:
- "Explore changes in equation and circuit with voltage and resistance sliders."
- "Explore changes in equation and circuit with sliders."
- "Explore changes in sim with voltage and resistance sliders."
- "Explore changes with voltage and resistance sliders."

Note, the words "Slider Controls" identifies the group of the two sliders, so removing it from the on-demand help text should be fine.

**Comment by terracoda** [comment]
@jessegreenberg, regarding the "Amount of dots", I think I am going to tweak the entire phrase. Just working on this in the design document.

**Comment by terracoda** [comment]
@jessegreenberg, I updated the [design document](https://docs.google.com/spreadsheets/d/15LuKXlzkh6xHpNbWC4ujtOfz1F7FJ7tj4oUF8Jvmj-k/edit#gid=0)

I marked changes in red.

**Summary of changes:**
- changed Scene Summary wording as EM suggested
- removed periods at end of the short list items in State of the Sim
- changed "represents" to "indicates" everywhere
- simplified "Ohm's Law Equation" heading to "The Equation". Reasoning, "Ohm's Law" is mentioned in h1 and Scene Summary times, and the shorter heading might balance nicely with "The Circuit" heading.
- Re-arranged the sentence describing resistance: [[Several dots]] in resistor indicate amount of resistance. 
- Changed on-demand help text for sliders to "Move sliders and explore changes in equation and circuit.", but I think I prefer a less directive approach like "Explore changes in sim as voltage and resistance change." I will email @ariel-phet which approach he likes best.

**Comment by terracoda** [comment]
@jessegreenberg, I've emailed @ariel-phet to get his opinion of the slider help text.

**Comment by ariel-phet** [comment]
@terracoda I am curious why does the slider help need to be "directive" at all? Could it be more something like "Voltage and resistance sliders allow changes to the equation and circuit"?

The PhET "ethos" has generally been to present options that are available, and more implicitly suggest an action. Have interviews suggested more directive/encouraging wording?

**Comment by terracoda** [comment]
@ariel-phet, I agree with the PhET design philosophy of "implicit scaffolding", and this is what I am trying to support in the examples in comment https://github.com/phetsims/ohms-law/issues/80#issuecomment-325983243

Interviews for _Balloons and Static Electricity (BASE)_ did find that the wording of help text can  hinder or support interaction in different ways. The grab interaction in BASE was particularly tricky, but much more complicated than the sliders in this sim.

Currently the number of examples we have of "interaction hints", and "on-demand help text" are limited. We are still learning about implicit scaffolding for the non-visual experience.

I like your suggestion: 
> Voltage and resistance sliders allow changes to the equation and circuit.

It neither explicitly tells the student to "explore the objects" or "move the sliders".  Let's go with this and see.

**Comment by terracoda** [comment]
@jessegreenberg, one small tweak to @ariel-phet's suggestion. I'm removing "the"
> Voltage and resistance sliders allow changes to equation and circuit.

I've updated the design document.

**Comment by ariel-phet** [comment]
@terracoda I know you agree :). I was more trying to explain my change in wording suggestion.

**Comment by terracoda** [comment]
@ariel-phet, I really like your change, and thanks for the explanation.

**Comment by jessegreenberg** [comment]
@terracoda are we ready to implement these changes? Please reassign the issue to me when you feel it is ready.

**Comment by terracoda** [comment]
@jessegreenberg, yes, I think it is safe to do so since I got Ariel's feedback. Reassigning to you.

**Comment by jessegreenberg** [comment]
@terracoda to keep track of changes in the design doc, I am going to revert text color to black once they are updated, let me know if that is problematic.

**Comment by jessegreenberg** [comment]
Changed the descriptions in the above commits, but  interested in looking into

> Current arrows relative description doesn't seem to update (stays at "tiny arrows represent...").

**Comment by jessegreenberg** [comment]
I think that the problem is just that the current arrows grow like 1 / R.  The max height of the arrows is 1243 view coordinates.  For the vast majority of voltage and resistance values, the arrows are less than 200 view coordinates. The arrows grow 80% in the last 40 ohms.

@emily-phet @terracoda how would you like to handle this? If we describe the size of the arrows relative to their max/min heights, the current behavior is correct.

**Comment by jessegreenberg** [comment]
Since the description changes were addressed I will close this issue and create a new one for the above question.
